byway/src/main/webapp/script.js

```diff
@@ -12,17 +12,38 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+var selected = [];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+/** Track the selected interests in an array according to their status. */
```

Comment:
Since javascript doesn't have typing, it would be useful to document what type of parameters you function expects. You can follow the style guide recommendations outlined here:
https://google.github.io/styleguide/jsguide.html#features-functions-top-level-functions

e.g. you could update this description like such:

```
/** 
 * Track the selected interests in an array according to their status. 
 * @param {Element} elem
 */
```

Reply:
Fixed